packages/block-editor/src/components/block-toolbar/index.js

```diff
@@ -79,6 +80,8 @@ export default function BlockToolbar( { hideDragHandle } ) {
 		}
 	);
 
+	// Account for the cases where the block toolbar is rendered within the
+	// header are and not contextually to the block.
```

Comment:
Was this a typo for:

```suggestion
	// header area and not contextually to the block.
```